shared/utils/src/roundValue.js

```diff
@@ -11,19 +11,20 @@ digits: number of digits to round to
 export function roundValue(value, digits) {
 	const v = Number(value)
 	if (Number.isInteger(v)) return v
-	const abs = Math.abs(v)
-	if (abs < 1 || abs > 9999) {
-		//Number() reverts positive values less than 10^21 to a whole number
-		return abs > 9999 ? v.toPrecision(digits) : Number(v.toPrecision(digits))
-	}
+	if (Math.abs(v) < 1) return Number(v.toPrecision(digits))
 	return Number(v.toFixed(digits))
 }
 
-export function roundValueAuto(value) {
+/** Rounds numbers to the appropriate decimal point
+ * if format is true, returns either a number or string in
+ * scientific notation.
+ */
+
+export function roundValueAuto(value, format = false) {
```

Comment:
roundValueAuto(value) will always add 2 decimals even for very large (absolute) numbers, which is not necessary. please note this as a TODO